### ISSUE TITLE
WhirlyGlobeMaplyComponent: No such file or directory -- While Making release build from xcode 14.2

### ISSUE THREAD

**Issue by Harshit10112** [mousebird/WhirlyGlobe#1586]
Recently migrate project from xcode 13.2 to 14.2 after that i am facing issue of no such file or directory  while making build 

mv: rename /Users/mindruby/Library/Developer/Xcode/DerivedData/GAV_Unit_Configurator_T3-avxdykxqvavbbjcbrdzblbfftuxd/Build/Intermediates.noindex/ArchiveIntermediates/Pilot/InstallationBuildProductsLocation/Applications/Pilot.app/Frameworks/WhirlyGlobeMaplyComponent.framework/WhirlyGlobeMaplyComponent-merged to /Users/mindruby/Library/Developer/Xcode/DerivedData/GAV_Unit_Configurator_T3-avxdykxqvavbbjcbrdzblbfftuxd/Build/Intermediates.noindex/ArchiveIntermediates/Pilot/InstallationBuildProductsLocation/Applications/Pilot.app/Frameworks/WhirlyGlobeMaplyComponent.framework/WhirlyGlobeMaplyComponent: No such file or directory

**Comment by sjg-wdw** [comment]
The best way to figure out if it's us is to build the AutoTester app release and see if it happens there.

**Comment by Harshit10112** [comment]
Yes , I have tried with AutoTester app and make try to run and build but its give the same issues for some file  like "No such file or directory" 
<img width="1396" alt="Screenshot 2023-04-12 at 6 39 09 PM" src="https://user-images.githubusercontent.com/130026558/231467456-d8657e50-c82a-4120-8ee4-1acb65412ad4.png">

**Comment by sjg-wdw** [comment]
Those are just missing resources.  You can remove those from the project and build.
Even so that's odd.  I thought we had a clean project.  Is this the main or develop branch?

**Comment by Harshit10112** [comment]
its develop branch
